Dockerfile

```diff
@@ -1,5 +1,13 @@
-FROM alpine:3.10
+FROM golang:1.15-alpine as builder
```

Comment:
I'm not using a multi-stage Docker build in this project. Goreleaser is used to generate binaries and Docker containers and publish a new release to the GitHub repo.

Reply:
okay cool. I wasn't aware of that. I typically prefer to build inside docker, but since you're building outside and copying in, I'll remove this from the PR